package.json

```diff
@@ -159,8 +159,8 @@
     "@types/react-router-dom": "^5.3.3",
     "@types/url-parse": "^1.4.8",
     "@types/uuid": "^9.0.1",
-    "@typescript-eslint/eslint-plugin": "^4.8.1",
-    "@typescript-eslint/parser": "^4.8.1",
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
```

Comment:
The `@typescript-eslint/eslint-plugin: "^6.0.0"` requires at least node v16, but `.nvmrc` contains `14.20.0`